### ISSUE TITLE
[BUG] [PyTorch] Expected is_sm80 || is_sm90 to be true, but got false.

### ISSUE THREAD

**Issue by jke94** [huggingface/autotrain-advanced#478]
### Prerequisites

- [X] I have read the [documentation](https://hf.co/docs/autotrain).
- [X] I have checked other issues for similar problems.

### Backend

Local

### Interface Used

CLI

### CLI Command

// Powershell command 

autotrain dreambooth `
--project-name "carlo_project" `
--model "runwayml/stable-diffusion-v1-5" `
--image-path "fine-tuning_images_carlo/" `
--prompt "photo of carlo person" `
--resolution 512 `
--batch-size 1 `
--num-steps 200 `
--fp16 `
--gradient-accumulation 4 `
--lr 1e-4

### UI Screenshots & Parameters

![image](https://github.com/huggingface/autotrain-advanced/assets/53972851/c5ec27e2-0204-452b-b807-f9a365373a99)

### Error Logs

[error.log](https://github.com/huggingface/autotrain-advanced/files/14071934/error.log)

## OS
Windows

## python version 
Python 3.10.9

## requirements.txt for virtual environment
absl-py==2.1.0
accelerate==0.25.0
aiofiles==23.2.1
aiohttp==3.9.1
aiosignal==1.3.1
albumentations==1.3.1
alembic==1.13.1
altair==5.2.0
annotated-types==0.6.0
anyio==3.7.1
arrow==1.3.0
async-timeout==4.0.3
attrs==23.2.0
autotrain-advanced==0.6.81
bitsandbytes==0.41.0
Brotli==1.1.0
cachetools==5.3.2
certifi==2023.11.17
charset-normalizer==3.3.2
click==8.1.7
cmaes==0.10.0
codecarbon==2.2.3
colorama==0.4.6
colorlog==6.8.2
contourpy==1.2.0
cycler==0.12.1
datasets==2.14.7
diffusers==0.21.4
dill==0.3.7
docstring-parser==0.15
einops==0.6.1
evaluate==0.3.0
exceptiongroup==1.2.0
fastapi==0.104.1
ffmpy==0.3.1
filelock==3.13.1
fonttools==4.47.2
frozenlist==1.4.1
fsspec==2023.10.0
fuzzywuzzy==0.18.0
google-auth==2.27.0
google-auth-oauthlib==1.2.0
gradio==3.41.0
gradio_client==0.5.0
greenlet==3.0.3
grpcio==1.60.0
h11==0.14.0
hf_transfer==0.1.5
httpcore==1.0.2
httpx==0.26.0
huggingface-hub==0.20.3
idna==3.6
imageio==2.33.1
importlib-metadata==7.0.1
importlib-resources==6.1.1
inflate64==1.0.0
invisible-watermark==0.2.0
ipadic==1.0.0
Jinja2==3.1.3
jiwer==3.0.2
joblib==1.3.1
jsonschema==4.21.1
jsonschema-specifications==2023.12.1
kiwisolver==1.4.5
lazy_loader==0.3
loguru==0.7.0
Mako==1.3.0
Markdown==3.5.2
markdown-it-py==3.0.0
MarkupSafe==2.1.4
matplotlib==3.8.2
mdurl==0.1.2
mpmath==1.3.0
multidict==6.0.4
multiprocess==0.70.15
multivolumefile==0.2.3
networkx==3.2.1
nltk==3.8.1
numpy==1.26.3
oauthlib==3.2.2
opencv-python==4.9.0.80
opencv-python-headless==4.9.0.80
optuna==3.3.0
orjson==3.9.12
packaging==23.1
pandas==2.2.0
peft==0.7.1
Pillow==10.0.0
protobuf==4.23.4
psutil==5.9.8
py-cpuinfo==9.0.0
py7zr==0.20.6
pyarrow==15.0.0
pyarrow-hotfix==0.6
pyasn1==0.5.1
pyasn1-modules==0.3.0
pybcj==1.0.2
pycryptodomex==3.20.0
pydantic==2.4.2
pydantic_core==2.10.1
pydub==0.25.1
Pygments==2.17.2
pyngrok==7.0.3
pynvml==11.5.0
pyparsing==3.1.1
pyppmd==1.0.0
python-dateutil==2.8.2
python-multipart==0.0.6
pytz==2023.3.post1
PyWavelets==1.5.0
PyYAML==6.0.1
pyzstd==0.15.9
qudida==0.0.4
rapidfuzz==2.13.7
referencing==0.32.1
regex==2023.12.25
requests==2.31.0
requests-oauthlib==1.3.1
responses==0.18.0
rich==13.7.0
rouge-score==0.1.2
rpds-py==0.17.1
rsa==4.9
sacremoses==0.0.53
safetensors==0.4.2
scikit-image==0.22.0
scikit-learn==1.3.0
scipy==1.12.0
semantic-version==2.10.0
sentencepiece==0.1.99
shtab==1.6.5
six==1.16.0
sniffio==1.3.0
SQLAlchemy==2.0.25
starlette==0.27.0
sympy==1.12
tensorboard==2.15.1
tensorboard-data-server==0.7.2
texttable==1.7.0
threadpoolctl==3.2.0
tifffile==2023.12.9
tiktoken==0.5.1
tokenizers==0.15.1
toolz==0.12.1
torch==2.1.2+cu118
torchaudio==2.1.2+cu118
torchvision==0.16.2+cu118
tqdm==4.65.0
transformers==4.37.0
trl==0.7.4
types-python-dateutil==2.8.19.20240106
typing_extensions==4.9.0
tyro==0.7.0
tzdata==2023.4
urllib3==2.1.0
uvicorn==0.22.0
websockets==11.0.3
Werkzeug==2.3.6
win32-setctime==1.1.0
xgboost==1.7.6
xxhash==3.4.1
yarl==1.9.4
zipp==3.17.0

**Comment by jke94** [comment]
Can someone help me by giving me some information about the following problem? Also I have reproduce the following bug with the same code in Google Colab.

**Comment by dpasch01** [comment]
Have the same exact issue when fine-tuning llm Mistral7b.

**Comment by jke94** [comment]
> Have the same exact issue when fine-tuning llm Mistral7b.

Hi @dpasch01! I have observed that the error does not happen when the argument "--fp16" is missing. Looking the message in different pull request in PyTorch Github repository. The message exception matches with the meaning. I keep follow finding a documentation in autotrain-advanced or huggingface to avoid this error. Because this argument is accepted as "True" or "False" in the AutoTrain Google Collab example.
